examples/Cargo.toml

```diff
@@ -114,6 +114,10 @@ path = "examples/z_get.rs"
 name = "z_forward"
 path = "examples/z_forward.rs"
 
+[[example]]
+name = "z_liveliness"
+path = "examples/z_liveliness.rs"
```

Comment:
You should add `required-features = ["unstable"]`.

Reply:
Done